src/dotnet-libman/dotnet-libman.csproj

```diff
@@ -0,0 +1,65 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\dir.props" />
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <RootNamespace>Microsoft.Web.LibraryManager.Tools</RootNamespace>
+    <NuspecFile>dotnet-libman.nuspec</NuspecFile>
+    <PackageType>DotNetTool</PackageType>
+    <Description>Command line tool for Library Manager</Description>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.CommandLineUtils.Sources" PrivateAssets="All" Version="$(MicrosoftExtensionsCommandLineUtilsSourcesPackageVersion)" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\LibraryManager\Microsoft.Web.LibraryManager.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Update="Resources.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>Resources.resx</DependentUpon>
+    </Compile>
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Update="Resources.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>Resources.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
+  </ItemGroup>
+
+
+  <!--
+  **********************************************************
+    For packing global tools we need to use the nuspec file
```

Comment:
Is this why we have to do the hack in the build script to cache the Nerdbank-generated version info?

Reply:
Yup. There was a bug in the `pack` which prevented packaging a global tool with ProjectReferences, by just running pack on the `.csproj`